src/Microsoft.SymbolStore.Client/SymbolLocator.cs

```diff
@@ -0,0 +1,169 @@
+﻿using FileFormats.PDB;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using FileFormats;
+
+namespace Microsoft.SymbolStore.Client
+{
+    public sealed class SymbolLocator
+    {
+        private static readonly string[] s_microsoftSymbolServerUrls = new string[]
+        {
+            "http://msdl.microsoft.com/download/symbols",
+            "https://nuget.smbsrc.net",
```

Comment:
I think I tracked it down. It appears to the endpoint for downloading nuget packages hosted by symbolsource.org. I doubt it should be in this list.